lib/routes.js

```diff
@@ -0,0 +1,15 @@
+var routes = {
+  method: "GET",
+  path:   "/version",
+  config: {
+    state: {
+      parse:      true,
```

Comment:
This value defaults to true, checkout `parse` under http://hapijs.com/api#route-options